### DISCUSSION TITLE
qtile xorg vs wayland?

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3740] [Q&A] [answered]
first question how ı use qtile wayland (ı'm idiot please give me a simple example)
and the future is xorg or wayland? if wayland ı need change my /config.py?

**Comment by m-col** [comment, marked as answer]
There are some docs on installation for wayland: http://docs.qtile.org/en/stable/manual/install/index.html#wayland

and a bit more docs on some other wayland-related things: http://docs.qtile.org/en/stable/manual/wayland.html

The way of starting qtile is different between the two. In X, there is the X server and it starts qtile as a window manager (e.g. from .xinitrc). In Wayland, you would start qtile directly and it handles all the same things the X server handles.

>  the future is xorg or wayland?

The answer is yes but the reality is that there is no rush, X continues to work fine, and Wayland still needs some work (particularly Qtile's support of wayland is a bit buggy).

>  if wayland ı need change my /config.py?

Nope, your config should work fine as-is. The only changes necessary would be if you are using X-specific widgets, such as `Systray`, which you can replace with `StatusNotifier`.
